Zend/zend_compile.c

```diff
@@ -5654,6 +5671,10 @@ static zend_type zend_compile_typename(
 			ZSTR_VAL(type_str));
 	}
 
+	if ((type_mask == MAY_BE_ANY) && allow_null) {
+		zend_error_noreturn(E_COMPILE_ERROR, "Type mixed cannot be nullable");
```

Comment:
Maybe indicate that you can't *explicitly* mark it as nullable because it is already implicitly nullable?

Reply:
So `Type mixed cannot be nullable because it is already implicitly nullable`? Or maybe `Type mixed cannot be nullable since mixed already includes null`? Hm, this version probably builds too much on the union type nature of `mixed`.

Reply:
(I went with `Type mixed cannot be marked as nullable since mixed already includes null` for now, but I'll change it if we settle on a message)